lib/renuo/bin-check/servant_thread.rb

```diff
@@ -29,7 +29,8 @@ def run_command
     def override_output
       output = @script_config.script_output ||= @result.output
       error_output = @script_config.script_error_output ||= @result.error_output
-      @result = Result.new(output, error_output, @result.exit_code)
+      @result = Result.new(output + @script_config.appended_output, error_output + @script_config.appended_error_output,
```

Comment:
Shouldn't `output` always be the *whole* output? Is it the `standard_output`? Does it include `error_output`? Maybe a renaming also for `error_output` would clear up things.